server/personality/personality.service.ts

```diff
@@ -95,7 +93,7 @@ export class PersonalityService {
         personality.claims = await Promise.all(personality.claims.map((claim) => {
             return {
                 ...claim,
-                ...claim.revisions[0]
+                ...claim.lastestRevision
```

Comment:
we should clean-up the query above, although this is working, we don't need to populate revisions the way we did before